check_process

```diff
@@ -1,6 +1,6 @@
 ;; Test complet
 	; pre-install
-		sudo git clone https://github.com/YunoHost/CI_package_check /home/CI_package_check
+		(mkdir -p /home/CI_package_check/ && touch /home/CI_package_check/analyseCI.sh)
```

Comment:
![](https://i.imgflip.com/4zg5tp.jpg)

Reply:
I made that change because all test result are Level 0 with the git clone. but I will roll back that point ^^